### PR TITLE
Add container credentials information

### PR DESCRIPTION
**Pull request by gicmo**
Add a section how to configure container authorization and link from the blueprint guide as well as the container upload guide.

### CODE REVIEW

**Comment by gicmo** [comment]
@miabbott let me know if that is clear enough.

**Review by miabbott — APPROVED** [review]
LGTM!

File: osbuild-composer/src/user-guide/container-auth.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Container registry credentials
+
+All communication with container registries is done by the `osbuild-worker`
+service. It can be configured via the `/etc/osbuild-worker/osbuild-worker.toml`
+configuration file. It is read only once at on service start, so the service
```
Comment (dvdhrm) [inline osbuild-composer/src/user-guide/container-auth.md:5]: Spurious `on` in: `It is read only once at [on] service start, [...]`

File: osbuild-composer/src/user-guide/container-auth.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+needs to be restarted after making any changes.
+
+The configuration file has a `containers` section with an `auth_file_path`
+field that is a string referring to a path of a`containers-auth.json(5)` file
```
Comment (dvdhrm) [inline osbuild-composer/src/user-guide/container-auth.md:9]: Missing space before `containers-auth.json(5)`.

File: osbuild-composer/src/user-guide/container-auth.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+to be used for accessing protected resources. An example configuration could
+look like this:
+
+```Toml
+[containers]
+auth_file_path = "/etc/osbuild-worker/containers-auth.json"
+```
+
+For detail information about the format of the authorization file itself,
```
Comment (dvdhrm) [inline osbuild-composer/src/user-guide/container-auth.md:18]: -detail +detailed
[-about +on]

**Comment by gicmo** [comment]
Fixes via #85

**Pull request merged**